gapic/ads-templates/%namespace/%name/%version/%sub/services/%service/client.py.j2

```diff
@@ -23,6 +23,12 @@ from google.auth.transport.grpc import SslCredentials             # type: ignore
 from google.auth.exceptions import MutualTLSChannelError          # type: ignore
 from google.oauth2 import service_account                         # type: ignore
 
+try:
```

Comment:
I suggest making this into a macro shared with the standard templates. See my other comment on the topic.

Reply:
I did take a look at how this could be done and it's not a trivial change.  Shared macros for the ads templates will be revisited in https://github.com/googleapis/gapic-generator-python/issues/1994.

Reply:
Fixed in https://github.com/googleapis/gapic-generator-python/pull/1999/commits/109edd462b49315450ca2d6672dc408069834bdc

I created `_shared_macros.j2` and used a symlink to include the macros in ads templates